docs/src/development/setup.md

```
@@ -279,15 +315,15 @@ Gold Digger uses Cargo features for conditional compilation:
 ```toml
 # Default features
-default = ["json", "csv", "ssl", "additional_mysql_types", "verbose"]
+default = ["json", "csv", "additional_mysql_types", "verbose"]
 
 # Individual features
-json = ["serde_json"]
-csv = ["csv"]
-ssl = ["mysql/native-tls"]
-ssl-rustls = ["mysql/rustls-tls"]
+json = []  # Enable JSON output format
+csv = []   # Enable CSV output format
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)